alpakaConfig.cmake

```diff
@@ -381,12 +381,12 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
             ELSE()
                 IF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
                     IF(CUDA_VERSION VERSION_EQUAL 8.0)
-                        IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.0)
```

Comment:
Could you please retest this by adding a ubuntu 16.04 (5.5) and an ubuntu 14.04 (5.4) build to the travis.yml temporarily?

Reply:
Thanks for adding the builds, but you have to change/remove the check to see that it fails due to the tuple error.

Reply:
What would be good is if we could find a way to build a GCC 5.3 with CUDA 8.0.
Nevertheless, we do locally on Hypnos.